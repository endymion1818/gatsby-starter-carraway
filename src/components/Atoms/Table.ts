import styled from 'styled-components'

const Table = styled.table`
  &,
  thead,
  tbody,
  tfoot,
  tr,
  th,
  td {
    /* reset */
    border: none;
    border-collapse: inherit;
    border-spacing: 0;
    border-color: inherit;
    vertical-align: inherit;
    text-align: left;
    font-weight: inherit;
    -webkit-border-horizontal-spacing: 0;
    -webkit-border-vertical-spacing: 0;
  }
`
export default Table
