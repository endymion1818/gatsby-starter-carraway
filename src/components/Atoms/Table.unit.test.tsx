import { cleanup, render } from '@testing-library/react'
import React from 'react'
import Table from './Table'

describe('<Table />', () => {
  beforeEach(cleanup)
  it('Should render', () => {
    const { container } = render(<Table />)
    expect(container).toMatchSnapshot()
  })
})
