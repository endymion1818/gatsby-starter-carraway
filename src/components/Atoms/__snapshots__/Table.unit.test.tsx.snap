// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table /> Should render 1`] = `
.c0,
.c0 thead,
.c0 tbody,
.c0 tfoot,
.c0 tr,
.c0 th,
.c0 td {
  border: none;
  border-collapse: inherit;
  border-spacing: 0;
  border-color: inherit;
  vertical-align: inherit;
  text-align: left;
  font-weight: inherit;
  -webkit-border-horizontal-spacing: 0;
  -webkit-border-vertical-spacing: 0;
}

<div>
  <table
    class="c0"
  />
</div>
`;
